sycl/test/basic_tests/context_platforms.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %t.out
+#include <CL/sycl.hpp>
+#include <iostream>
+
+using namespace cl::sycl;
+
+int main(int argc, char const *argv[]) {
+  vector_class<platform> platforms = platform::get_platforms();
+  if (platforms.size() < 2) {
+    std::cout << " This test requires at least 2 SYCL platforms to be present "
+                 "on the system. Skipping it"
+              << std::endl;
+    return 0;
+  }
+  auto AsyncHandler = [](exception_list ES) {
+    for (auto &E : ES) {
+      std::rethrow_exception(E);
+    }
+  };
+  queue Q(AsyncHandler);
+  try {
+    Q.submit([&](handler &CGH) {
```

Comment:
You don't need `queue` to create `context` - only the code within `submit` method is needed without actual submitting